react-events/my-app/src/CustomButton.tsx

```diff
@@ -0,0 +1,21 @@
+type Props = {
+  text: string;
+  color: string;
+  onCustomClick: Function;
```

Comment:
As you get more comfortable with TS you'll want to make this more specific. Instead of accepting any function, you'll want to only accept one that accepts the correct parameters and has the correct return type.

This is the syntax you would use:
`. onCustomClick: (color: string, text: string) => void;`
which says that `onCustomClick` is a function that has two string parameters and returns void (that is, doesn't return anything).